### PR TITLE
refactor: move log-file to be under --log

### PR DESCRIPTION
**Pull request by josedonizetti**
### 1. Explain what the PR does]

Fix https://github.com/aquasecurity/tracee/issues/2908

### 2. Explain how to test it

```
tracee --log file:/tmp/test.log
cat /tmp/test.log
```

### 3. Other comments

This refactoring is done now, because it simplifies the implementation of actions per events for the policies task: https://github.com/aquasecurity/tracee/issues/2665

### CODE REVIEW

File: pkg/cmd/flags/logger.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ Possible options:
   --log warn                     | warning log level
   --log error                    | error log level
   --log panic                    | panic log level
+  --file:/path/to/file           | write the logs to a specified file. create/trim the file if exists (default: stderr)
```
Comment (yanivagman) [inline pkg/cmd/flags/logger.go:24]:
```suggestion
  --log file:/path/to/file           | write the logs to a specified file. create/trim the file if exists (default: stderr)
```

**Review by yanivagman — APPROVED** [review]
LGTM

File: pkg/cmd/flags/logger.go [hunk 2]
```diff
@@ -21,6 +21,7 @@ Possible options:
   --log warn                     | warning log level
   --log error                    | error log level
   --log panic                    | panic log level
+  --log file:/path/to/file       | write the logs to a specified file. create/trim the file if exists (default: stderr)
```
Comment (rafaeldtinoco) [inline pkg/cmd/flags/logger.go:24]: Other options are mutually exclusive, this one is not. Somehow the user should be told that.

Comment (josedonizetti) [reply pkg/cmd/flags/logger.go:24]: Makes sense, I considered it, but as `aggregate` is also like `file` and can be used with other flags, I didn't mind. Changing to what Yaniv suggested.

**Review by geyslan — APPROVED** [review]
Put a worry about file closing.

Overall this LGTM.

File: pkg/cmd/flags/logger.go [hunk 3]
```diff
@@ -14,34 +14,49 @@ func logHelp() string {
...
 Possible options:
-  --log aggregate                | turns log aggregation on, delaying output (default is off)
-  --log aggregate:interval       | turns log aggregation on, delaying output to every 'interval' (s, m)
-  --log debug                    | debug log level
-  --log info                     | information log level (default)
-  --log warn                     | warning log level
-  --log error                    | error log level
-  --log panic                    | panic log level
+  --log aggregate                     | turns log aggregation on, delaying output (default is off)
+  --log aggregate:interval            | turns log aggregation on, delaying output to every 'interval' (s, m)
```
[the remark this answers is not in the document]
Comment (josedonizetti) [reply pkg/cmd/flags/logger.go:18]: Good one!

**Pull request merged**